screens/ProductDetailsScreen.tsx

```diff
@@ -311,6 +311,7 @@ export default function ProductDetailsScreen({ route, navigation }) {
     }
   };
   console.log("auth?.currentUser?.email" + auth?.currentUser?.email);
+  console.log(`current user: ${JSON.stringify(auth.currentUser)}`);
```

Comment:
Are these logs still useful? If not we should remove them

Reply:
I don't recall modifying this file, and I don't want to get into general code base cleaning and refactoring since this PR is focused on sign in. I'm sure there are misplaced / not useful console logs all over the place but I don't think this PR is where I address all of those.